### PR TITLE
Remove lazy imports

### PR DESCRIPTION
**Pull request by ralsina**
Remove LazyImports class.

Yet another step in my quest to make rst2pdf's codebase less weird :-)

### CODE REVIEW

File: rst2pdf/opt_imports.py [hunk 1]
```diff
@@ -69,42 +69,5 @@
...
-
-    def _load_svg2rlg(self):
-        if six.PY2:
-            import svg2rlg
-            return svg2rlg
-        return None
-
-LazyImports = LazyImports()
+import pdfrw as pdfinfo
+from PIL import Image as PILImage
```
Comment (akrabat) [inline rst2pdf/opt_imports.py]: What is the benefit of importing these two within opt_imports compared to importing directly in `image.py`? 

Or, to put it another way, should there be a `try` around these imports?

Comment (ralsina) [reply rst2pdf/opt_imports.py]: Just wanted to make the change minimal.

They don't need a try/except because we are really requiring them now :-)

Comment (akrabat) [reply rst2pdf/opt_imports.py]: Got it. My gut feeling is that they shouldn't be in opt_imports as they aren't optional. Your call though as you know this better than I do.

**Comment by ralsina** [comment]
I don't want to have any optional imports at all, so in a followup branch I
can just get rid of that file

**Comment by akrabat** [comment]
> I don't want to have any optional imports at all, so in a followup branch I can just get rid of that file

Works for me! 👍

**Pull request merged**